src/follow_the_gap/__main__.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+from pathlib import Path
```

Comment:
```suggestion
#!/usr/bin/env python3

import argparse
import sys

from pathlib import Path
```
Just a stylistic nit: let your code *breathe* a bit.